src/Namotion.Reflection/Namotion.Reflection.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard2.0;net40;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard1.0;netstandard2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Why are you changing this? 
This is a major breaking change and netstandard2.0 is enough for .NET Core 3.1

Reply:
Yeah that's just my mistake. :')